cypress/integration/common/kiali_homepage.ts

```diff
@@ -0,0 +1,13 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+import { Then } from "cypress-cucumber-preprocessor/steps";
+
+const url = "https://kiali-istio-system.apps.pmdown4.0216-eg3.fw.rhcloud.com"
```

Comment:
If you set `baseUrl` through an env var you shouldn't need to set this explicitly: https://docs.cypress.io/guides/references/best-practices#Setting-a-global-baseUrl. `cy.visit(url)` then becomes `cy.visit("/")`.